backend/controllers/caseNote.js

```diff
@@ -178,6 +178,30 @@ const caseNoteFactory = (prisonApi, caseNotesApi) => {
       // has been handled by the above validations
     }
 
+    if (errors.length === 0) {
```

Comment:
Might be neater pulling this out into a function

Reply:
I tried that, and it didn't improve readability for me due to the way it returned the error(s) or threw one.

Maybe you had a specific impl in mind that might be better?